### ISSUE TITLE
cache file required

### ISSUE THREAD

**Issue by fidelram** [rfechtner/pypairs#6]
I had an issue with pypairs when running:

```PYTHON
from pypairs import pairs, datasets
training_data = datasets.leng15(mode='sorted')
```

The program hanged after a while, giving a warning that it could not write into a `cache` folder.

The issue was solved by creating a `cache` folder in my working directory.

Maybe you could create this folder automatically.

**Comment by rfechtner** [comment]
Hi @fidelram,

Thanks for bringing that up. I just implemented the caching to improve speed. 

I will make sure to catch that exception and allow processing without a cache dir, and include a automatic generation. 

Cheers

Ron

**Comment by fidelram** [comment]
Great! Thanks

**Comment by rfechtner** [comment]
Should not be resolved in PyPairs v3.0.11
